### DISCUSSION TITLE
Cannot declare class WordPressValetDriver

### DISCUSSION THREAD

**Discussion by iantearle** [laravel/valet#1283] [Q&A]
Trying to use WordPressValet from the valet Github repo. If I add to the drivers folder I get this Fatal Error:

``` Cannot declare class WordPressValetDriver, because the name is already in use in /Users/iantearle/.config/valet/Drivers/WordPressValetDriver.php on line 0```

I also have WordPressProxyValetDriver, but even without this I get the above error.

Could I have a secondary driver folder? If so how would I find it?

**Comment by Gkiokan** [comment]
I had to update my valet for the php@8.2 support and accidently updated to much.  
However, it seems there is a breaking change with the Driver Directory in the latest versions.  
 
To fix this you need to Delete everything in your Drivers folder and it will work again.  
So go to `~/.config/valet/Drivers` and call `rm -rf ./*` 
 
 In my case I had a vendor folder which got loaded and triggered this issue.

**Reply by drbyte** [reply]
There were changes to the driver syntax. See https://github.com/laravel/valet/blob/master/UPGRADE.md

**Reply by Gkiokan** [reply]
If you come from an early Valet v3 release, the Drivers folder had another vendor Folder inside which end up exactly the redeclaration issue above.
This seems to be removed in the latest Valet release, so you have to clean it up manually as the installation doesn't take care of it.  

And of course, all Custom Drivers has to match the new Syntax which is annoying but meh, at least it isn't a big change.
